Lib/test/test_call.py

```diff
@@ -665,5 +666,52 @@ def __call__(self, *args):
                 self.assertEqual(expected, wrapped(*args, **kwargs))
 
 
+class A:
+    def method(self, x, y):
+        return x + y
+
+    @staticmethod
+    def static_method():
+        pass
+
+    @staticmethod
+    def positional_only(arg, /):
+        pass
+
+@cpython_only
+class TestErrorMessagesUseQualifiedName(unittest.TestCase):
+
+    @contextlib.contextmanager
+    def check_raises_exact(self, message, exception_type=TypeError):
```

Comment:
Since you're not using `exception_type`, you can eliminate that argument. If you're worried about the name, you could call it `check_raises_type_error`.